contrib/tests/actions/chains/test_quickstart_rules.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 vars:
-        tested_rule: examples.webhook_file
+        tested_rule: examples.examples.webhook_file
```

Comment:
Yeah, I went for that change with the PR and the migration had issues because the name changes before and after the upgrade. So either we should wait until 0.10 release to make this change or know that there might be stale rules around since we renamed.